client/src/partials/debtor_groups/debtor_groups.html

```diff
@@ -0,0 +1,289 @@
+                    name="phone"
+                    placeholder="{{ 'COLUMNS.PHONE' | translate }}..."
+                    ng-model="DebtorGroupCtrl.debtorGroup.phone">
+                </div>
+
+                <!-- email  -->
+                <div class="form-group">
+                  <label class="control-label">
+                    {{ "COLUMNS.EMAIL" | translate }}
+                  </label>
+                  <input
+                    class="form-control"
+                    name="email"
+                    placeholder="{{ 'COLUMNS.EMAIL' | translate }}..."
+                    ng-model="DebtorGroupCtrl.debtorGroup.email">
+                </div>
+
+                <!-- debtor group location  -->
+                <bh-location-select
+                  id="origin-location-id"
+                  validation-trigger="ActionForm.$submitted"
+                  location-uuid="DebtorGroupCtrl.debtorGroup.location_id">
+                </bh-location-select>
+              </fieldset>
+
+              <!-- note  -->
+              <div class="form-group">
+                <label class="control-label">
+                  {{ "COLUMNS.NOTE" | translate }}
+                </label>
+                <textarea
+                  class="form-control"
+                  name="note"
+                  placeholder="{{ 'COLUMNS.NOTE' | translate }}..."
+                  ng-model="DebtorGroupCtrl.debtorGroup.note">
+                </textarea>
+              </div>
+
+            </div>
+
+            <div class="panel-footer">
+
+              <button class="btn btn-default pull-right" type="submit" data-method="submit">
+                {{ "FORM.SUBMIT" | translate }}
+              </button>
+              <button class="btn btn-default" type="button" ng-click="DebtorGroupCtrl.cancel()" data-method="cancel">
+                {{ "FORM.CANCEL" | translate }}
+              </button>
+
+              <div class="clearfix"></div>
+
+              <!-- error feedback area -->
+              <div ng-if="DebtorGroupCtrl.state.errored && ActionForm.$submitted"
+                class="text-danger" data-role="feedback" style="margin-top:10px;">
+                <i class="glyphicon glyphicon-remove-sign"></i> {{ "FORM.HAS_ERRORS" | translate }}
+              </div>
+
+            </div>
+          </form>
+        </div>
+
+        <!-- success feedback -->
+        <div class="alert alert-success" ng-switch-when="success">
+          <h4 id="update_success" ng-if="DebtorGroupCtrl.state.updated">
+            <i class="glyphicon glyphicon-ok-sign"></i> {{ "DEBTOR_GRP.UPDATE_SUCCESS" | translate }}
+          </h4>
+          <h4 id="create_success" ng-if="DebtorGroupCtrl.state.created">
+            <i class="glyphicon glyphicon-ok-sign"></i> {{ "DEBTOR_GRP.CREATE_SUCCESS" | translate }}
+          </h4>
+        </div>
+
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Templates  -->
+<script type="text/ng-template" id="accounts.tpl">
+  <a>
```

Comment:
This comment is just a remark.  Here is something I learned about `typeahead`s yesterday:
```html
<script type="text/ng-template" id="accounts.tpl">
  <a>
      <span ng-bind-html="match.label | uibTypeaheadHighlight:query">
        <strong>{{ match.model.account_number}}</strong>
        <em>{{ match.model.account_txt }}</em>
       </span>
  </a>
</script>
```
This will allow the typahead to highlight the content as you type.  Pretty cool!  Try it out and see if you like it.